src/Pages/Music/Album/Albumid_Int.elm

```diff
@@ -154,10 +154,18 @@ view model =
                 column [ Element.height fill, Element.width fill, Background.color Colors.sidebar ]
                     [ row [ Element.height (fillPortion 1), Element.width fill, Background.color (Element.rgba255 50 53 55 1), Element.htmlAttribute (Html.Attributes.class "card-parent"), padding 30 ]
                         [ column [ Element.width (px 250), Element.height (px 250), Element.htmlAttribute (Html.Attributes.class "card-parent") ]
-                            [ image [ Element.height (fill |> maximum 230), Element.width (fillPortion 2 |> maximum 230) ]
-                                { src = crossOrigin "http://localhost:8080" [ "image", percentEncode album.thumbnail ] []
-                                , description = "Thumbnail"
-                                }
+                            [ case album.thumbnail of
```

Comment:
Hrm, the fallback logic might be worthwhile to move to it's own function, as we seem to use it everywhere. With enough variables it might be able to nicely combine that code and still be flexible enough? Just and idea.